### PR TITLE
HAAR-889: List roles of a user. Updated security modules

### PR DESCRIPTION
**Pull request by ravmoj**
List roles of a user. Updated security modules

### CODE REVIEW

File: integration-tests/mockApis/auth.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ const stubUserMe = ({
     body: { username, firstName, lastName, name, activeCaseLoadId },
   })
 
-const stubUserMeRoles = (roles) => getFor({ urlPattern: '/auth/api/user/me/roles', body: roles })
+const stubUserMeRoles = (roles) => getFor({ urlPattern: '/users/me/roles', body: roles })
```
Comment (karenmillermoj) [inline integration-tests/mockApis/auth.js:127]: Should this stub be in mockApis/manageusers.js

Comment (karenmillermoj) [reply integration-tests/mockApis/auth.js:127]: Is this used anywhere?

Comment (ravmoj) [reply integration-tests/mockApis/auth.js]: Moved to manageusers.js

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**